src/static_analysis/code.rs

```diff
@@ -62,11 +62,11 @@ pub fn analysis<S: AsRef<str>>(
 
     let handles: Vec<_> = (0..config.get_threads())
         .map(|_| {
-            let thread_manifest = manifest.clone();
-            let thread_files = files.clone();
-            let thread_rules = rules.clone();
-            let thread_vulns = found_vulns.clone();
-            let thread_dist_folder = dist_folder.clone();
+            let thread_manifest = Arc::clone(&manifest);
```

Comment:
Hmm interesting change, any particular reason?

Reply:
To be honest, clippy told me that was preferred :laughing: